usaspending_api/transactions/delta_models/transaction_search.py

```diff
@@ -462,10 +462,14 @@
         SAA.abbreviation AS awarding_subtier_agency_abbreviation,
         SFA.abbreviation AS funding_subtier_agency_abbreviation,
         COALESCE(transaction_fabs.awarding_office_code, transaction_fpds.awarding_office_code) AS awarding_office_code,
-        COALESCE(transaction_fabs.awarding_office_name, transaction_fpds.awarding_office_name) AS awarding_office_name,
-        COALESCE(transaction_fabs.funding_office_code, transaction_fpds.funding_office_code) AS funding_office_code,
-        COALESCE(transaction_fabs.funding_office_name, transaction_fpds.funding_office_name) AS funding_office_name,
-
+        CASE
```

Comment:
Since you are doing a `LEFT JOIN` to the `office` table down below you should be able to change this to simply use `awarding_office.office_name` instead of a `CASE` statement. A `NULL` value for FPDS / FABS office_code will result in `NULL` from the LEFT JOIN anyways.